.github/workflows/stale.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '39 7 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        days-before-stale: -1
+        days-before-close: -1
+        days-before-issue-stale: 90
+        days-before-issue-close: 90
+        stale-issue-message: >
+          This issue has been automatically marked as stale because it has not had
+          recent activity. It will be closed if no further activity occurs. Thank you
+          for your contributions.
+        close-issue-message: >
+          This issue has been automatically closed because it has not had recent
+          activity. Please comment "/reopen" to reopen it.
+        stale-issue-label: 'lifecycle/stale'
```

Comment:
NIT: Only this value has single quote, feel free to skip my comment if the behavior doesn't have a difference with/without quote.